src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreResult.cs

```diff
@@ -8,6 +8,11 @@ namespace NuGet.PackageManagement
 {
     public class PackageRestoreResult
     {
+        public PackageRestoreResult(bool restored, IEnumerable<PackageIdentity> restoredPackages, AuditCheckResult auditCheckResult)
+            : this(restored, restoredPackages)
+        {
+        }
```

Comment:
`auditCheckResult` isn't being used (not getting saved into the property).

If you `#nullable enable` this 30 line file, I'm pretty sure the compiler would tell you about this mistake.